py3-pyserial.yaml

```diff
@@ -0,0 +1,110 @@
+# Generated from https://pypi.org/project/pyserial/
+package:
+  name: py3-pyserial
+  version: "3.5"
+  epoch: 0
+  description: Python Serial Port Extension
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    provider-priority: "0"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-supported-build-base-dev
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 0e7634747568547b8a7f9fd0c48ed74f16af4b23
+      repository: https://github.com/pyserial/pyserial
+      tag: v${{package.version}}
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    pipeline:
+      - name: Python Build
+        uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+    dependencies:
+      provides:
+        - py3-${{vars.pypi-package}}
+      provider-priority: ${{range.value}}
+    test:
+      environment: {}
+      pipeline:
+        - name: Import Test
+          uses: python/import
+          with:
+            import: ${{vars.import}}
+            python: python${{range.key}}
+
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
+    dependencies:
+      runtime:
+        - py3.10-${{vars.pypi-package}}
+        - py3.11-${{vars.pypi-package}}
+        - py3.12-${{vars.pypi-package}}
+        - py3.13-${{vars.pypi-package}}
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python3.10
+            import: ${{vars.import}}
+
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}-bin
+    description: Executable binaries for ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      runtime:
+        - py${{range.key}}-${{vars.pypi-package}}
+      provides:
+        - py3-${{vars.pypi-package}}-bin
+        - py3-${{vars.pypi-package}}
```

Comment:
I don't think we want the `-bin` packages providing this. We've done that in cases where we are migrating from a single `py3-foo` that provided both lib and bin in one, so that installing `py3-foo` still brought in commands like it did before . But this package is new, so that's not an issue here.

But also - weren't we going to drop the `py3-*` provides for  *new* `py3.x-*` packages anyway? Not a blocker if you want to keep them for now, thanks for opening the issue on that.